src/array.jl

```diff
@@ -304,8 +290,22 @@ function droplevels!(A::CatArray)
     levels!(A, intersect(levels(A.pool), index(A.pool)[found]))
 end
 
-ordered(A::CatArray) = ordered(A.pool)
-ordered!(A::CatArray, ordered) = ordered!(A.pool, ordered)
+"""
+    isordered(A::CategoricalArray)
+    isordered(A::CategoricalPool)
```

Comment:
Let's not mention `CategoricalPool` here, as it's a semi-private API. Anyway, I have larger docstring changes locally.